x-pack/plugins/reporting/server/routes/lib/jobs_query.ts

```diff
@@ -21,17 +21,20 @@ import { ReportApiJSON, ReportSource } from '../../../common/types';
 import { statuses } from '../../lib/statuses';
 import { Report } from '../../lib/store';
 import { ReportingUser } from '../../types';
+import { runtimeFields, runtimeFieldKeys } from '../../lib/store/runtime_fields';
 
 const defaultSize = 10;
 const getUsername = (user: ReportingUser) => (user ? user.username : false);
 
 function getSearchBody(body: SearchRequest['body']): SearchRequest['body'] {
   return {
     _source: {
-      excludes: ['output.content'],
+      excludes: ['output.content', 'payload.headers'],
```

Comment:
This is a spot-fix improvement to receive fewer bytes over the connection from ES to the Kibana server, when querying jobs for the Stack Management UI